spec/dealership_spec.rb

```diff
@@ -0,0 +1,102 @@
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe "Dealership" do
+  before(:each) do
+    @dealership = Dealership.new("Acme Auto", "123 Main Street")
+    @car_1      = Car.new("Ford Mustang", 1500, 36)
+    @car_2      = Car.new("Toyota Prius", 1000, 48)
+    @car_3      = Car.new("Toyota Tercel", 500, 48)
+    @car_4      = Car.new("Chevrolet Bronco", 1250, 24)
+  end
+
+  it "exists" do
+    expect(@dealership).to be_a Dealership
+  end
+
+  it "has attributes" do
+    expect(@dealership.name).to eq("Acme Auto")
+    expect(@dealership.address).to eq("123 Main Street")
+    expect(@dealership.inventory).to eq([])
+  end
+
+  it "can add cars to its inventory" do
+    @dealership.add_car(@car_1)
+    @dealership.add_car(@car_2)
+
+    expect(@dealership.inventory).to eq([@car_1, @car_2])
+  end
+
+  it "can count how many cars it has in inventory " do
```

Comment:
I think there was a expect for `inventory_count` to start as `0`

```
pry(main)> dealership.inventory_count
#=> 0
```
I am being picky 😉